### PR TITLE
BAEL-9

### PR DESCRIPTION
**Pull request by Thoughtscript**

### CODE REVIEW

File: spring-boot-servlet/src/main/java/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
+Main-Class: com.jspbaeldung.ApplicationMain
```
Comment (KevinGilmore) [inline spring-boot-servlet/src/main/java/META-INF/MANIFEST.MF]: not the correct class?

File: spring-boot-servlet/src/main/java/com/baeldung/props/Constants.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package com.baeldung.props;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Properties;
+
+public final class Constants {
```
Comment (KevinGilmore) [inline spring-boot-servlet/src/main/java/com/baeldung/props/Constants.java]: Is this class still being used? I didn't see any references to it or any of the public properties.

File: spring-boot-servlet/src/main/java/com/baeldung/props/PropertyLoader.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package com.baeldung.props;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertyLoader {
```
Comment (KevinGilmore) [inline spring-boot-servlet/src/main/java/com/baeldung/props/PropertyLoader.java]: Is this class still used, other than by the Constants class?
If you decide that you no longer need the Constants class, then do you still need this one?

File: spring-boot-servlet/src/main/java/com/baeldung/props/PropertyLoader.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        Properties prop = new Properties();
+        InputStream input = null;
+        try {
+          input = getClass().getResourceAsStream(file);
+          prop.load(input);
+        } catch (IOException ex) {
+          if (input != null) {
+            input.close();
+          }
+        } finally {
```
Comment (KevinGilmore) [inline spring-boot-servlet/src/main/java/com/baeldung/props/PropertyLoader.java]: In Eclipse, I see a compiler warning (finally block does not complete normally).
May be a moot point if you no longer need this class.

File: spring-boot-servlet/pom.xml [hunk 5]
```diff
@@ -0,0 +1,55 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.baeldung</groupId>
+    <artifactId>ServletReg</artifactId>
```
Comment (Thoughtscript) [inline spring-boot-servlet/pom.xml]: Should have caught that - thanks.

File: spring-boot-servlet/src/main/java/com/baeldung/props/Constants.java (hunk 2 above)
Comment (Thoughtscript) [inline spring-boot-servlet/src/main/java/com/baeldung/props/Constants.java]: This exists simply to illustrate how to set this up - otherwise no.

**Pull request closed without merging**